### PR TITLE
[SIS-032] Wifi warning and unsaved changes overlays

### PR DESCRIPTION
**Pull request by BakerWJ**
# Description

Created overlays for wifi warnings and unsaved changes.

Pasted Graphic

image

Ticket #32

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My PR title is formatted as [SIS-\<issue number\>] \<issue name\> (eg. [SIS-010] Create UI button)
- [x] I have linked my issue/ticket on the project board to this PR
- [x] I have sent my PR to #team-sistema-prs on Slack and requested two people to review
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [ ] I have tested that my feature works as expected on Android and iOS. If you are a Windows dev, please request a Mac dev to run your branch. If it's a front-end change, I have attached screenshots of what the result looks like on both platforms.

### CODE REVIEW

**Review by ramyzhang — CHANGES_REQUESTED** [review]
Can you add more padding between the cloud icon and the text in the wifi warning? As well as in between text lines and the buttons on the bottom for both overlays? Otherwise looks good, thank you!

File: src/Components/Overlay.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (yuemily24) [inline src/Components/Overlay.js]: Don't think this import is needed?

File: src/Components/SistemaButton.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (yuemily24) [inline src/Components/SistemaButton.js]: Same thing

File: src/Components/SistemaButton.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { TouchableOpacity, StyleSheet, Text } from 'react-native';
+
+const Button = props => {
```
Comment (yuemily24) [inline src/Components/SistemaButton.js]: Rename to `SistemaButton`? Just for clarity

File: src/Components/SistemaButton.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    borderWidth: 0.77,
+    borderRadius: 6,
+    borderColor: '#453E3D',
+    paddingHorizontal: 12,
+    paddingVertical: 6,
+    alignSelf: 'center'
+  }
+});
+
+export default Button;
```
Comment (yuemily24) [inline src/Components/SistemaButton.js]: Don't forget to change the default export if gets changed

File: src/editor/screens/UnsavedChangesOverlay.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```
Comment (yuemily24) [inline src/editor/screens/UnsavedChangesOverlay.js]: Record scratch

**Review by yuemily24 — CHANGES_REQUESTED** [review]
Very minor changes, otherwise looks good to me!

File: src/editor/screens/UnsavedChangesOverlay.js (hunk 5 above)
Comment (BakerWJ) [reply src/editor/screens/UnsavedChangesOverlay.js]: [IM A BOOMER IM SORRY](https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file)

Comment (yuemily24) [reply src/editor/screens/UnsavedChangesOverlay.js]: LOL all g

**Comment by BakerWJ** [comment]
@ramyzhang here is picture, added a bit more margin than the og one.

image

**Review by ramyzhang — APPROVED** [review]
eggcellent pls do conflict resolutions and this will be good 2 go! tyty

**Comment by BakerWJ** [comment]
i love how github says there are merge conflicts, i go to fix on my laptop and BOOOM no conflicts

**Pull request merged**